### ISSUE TITLE
`type IOError has no field code`

### ISSUE THREAD

**Issue by KristofferC** [JuliaCloud/AWS.jl#506]
Looking at this log: https://s3.amazonaws.com/julialang-reports/nanosoldier/pkgeval/by_hash/3f030a7_vs_f23fc0d/Zarr.1.7.0-rc2-c249b791179.log it looks like there is some missing error handling:

```
Minio S3 storage: Error During Test at /home/pkgeval/.julia/packages/Zarr/2V5d9/test/storage.jl:79
  Got exception outside of a @test
  type IOError has no field code
  Stacktrace:
    [1] getproperty(x::HTTP.IOExtras.IOError, f::Symbol)
      @ Base ./Base.jl:42
    [2] macro expansion
      @ ~/.julia/packages/Retry/vS1bg/src/repeat_try.jl:110 [inlined]
    [3] submit_request(aws::MinioConfig, request::AWS.Request; return_headers::Nothing)
      @ AWS ~/.julia/packages/AWS/nUGJF/src/utilities/request.jl:85
    [4] (::AWS.RestXMLService)(request_method::String, request_uri::String, args::Dict{String, Any}; aws_config::MinioConfig, feature_set::AWS.FeatureSet)
      @ AWS ~/.julia/packages/AWS/nUGJF/src/AWS.jl:284
    [5] create_bucket(Bucket::String; aws_config::MinioConfig)
      @ Zarr.S3 ~/.julia/packages/AWS/nUGJF/src/services/s3.jl:478
    [6] create_bucket(Bucket::String)
      @ Zarr.S3 ~/.julia/packages/AWS/nUGJF/src/services/s3.jl:478
```

**Comment by omus** [comment]
Should be addressed by both #501 and #494. @mattBrzezinski looks like we just need a release for these.

**Comment by KristofferC** [comment]
:+1:
